Library/Formula/sip.rb

```diff
@@ -13,7 +13,8 @@ class Sip < Formula
 
   head "http://www.riverbankcomputing.co.uk/hg/sip", :using => :hg
 
-  depends_on :python => :recommended
+  option "without-python", "Build without python2 support"
```

Comment:
I wonder can we make it `depends_on :python => [:recommended, :run]`? It seems to be better regarding DSL.